Sources/Features/Breadcrumb/BacktraceBreadcrumbFileHelper.swift

```diff
@@ -1,30 +1,26 @@
 import Foundation
-import Cassette
 
 enum BacktraceBreadcrumbFileHelperError: Error {
     case invalidFormat
 }
 
 @objc class BacktraceBreadcrumbFileHelper: NSObject {
 
-    /*
-     The underlying library CASQueueFile assigns a minimum of 4k (filled with zeroes).
-     Since we know that space will be allocated (and uploaded) anyways, set it as the minimum.
-     */
     private static let minimumQueueFileSizeBytes = 4096
 
     private let maximumIndividualBreadcrumbSize: Int
     private let maxQueueFileSizeBytes: Int
-    private let queue: CASQueueFile
+    
+    private let queue: Queue<Any>
+    private let breadcrumbLogURL: URL
 
-    /** CASQueueFile is not thread safe, so all interactions with it should be done synchronously through this DispathQueue */
     private let dispatchQueue = DispatchQueue(label: "io.backtrace.BacktraceBreadcrumbFileHelper@\(UUID().uuidString)")
 
     public init(_ breadcrumbSettings: BacktraceBreadcrumbSettings) throws {
-        self.queue = try CASQueueFile.init(path: breadcrumbSettings.getBreadcrumbLogPath().path)
-
+        
+        self.breadcrumbLogURL = try breadcrumbSettings.getBreadcrumbLogPath()
```

Comment:
what if the file doesn't exist? we don't create it anywhere. We always write to a file. How about creating a start method that will do startup logic?

Reply:
the file will always exist;
`getBreadcrumbLogPath()` method will create a file if doesn't exist

Reply:
there is no good way to do this since BacktraceBreadcrumbSettings forces us to (try/unwrap), if for any reason the file is nil the files operations methods will catch the error and warn BacktraceLogger.